pages/linux/arch-chroot.md

```diff
@@ -0,0 +1,24 @@
+# arch-chroot
+
+> Enhanced `chroot` command for Arch Linux installation.
```

Comment:
Maybe this is clearer?
```suggestion
> Enhanced `chroot` command for Arch Linux systems.
```

Reply:
I think no, because this command is a helper for the Arch Linux installation process, not for users of the OS.

Reply:
Ah, gotcha. Then I'd suggest saying precisely that:
```suggestion
> Enhanced `chroot` command to help in the Arch Linux installation process.
```